tcms/testruns/templates/testruns/get.html

```diff
@@ -127,32 +127,32 @@ <h2 class="card-pf-title">
                                         <div class="input-group-btn">
                                             <button type="button" class="btn btn-default dropdown-toggle" id="input-filter-button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Name <span class="caret"></span></button>
                                             <ul class="dropdown-menu">
-                                                <li class="selected"><a href="#">Name</a></li>
-                                                <li><a href="#">Type</a></li>
-                                                <li><a href="#">Color</a></li>
+                                                <li class="selected"><a href="#">{% trans "Name" %}</a></li>
+                                                <li><a href="#">{% trans "Priority" %}</a></li>
+                                                <li><a href="#">{% trans "Category" %}</a></li>
```

Comment:
Do not get too much concerned with filters ATM. There is a separate task for his functionality b/c I have the feeling it will need more than meets the eye.